java/code/src/com/suse/oval/OVALCleaner.java

```diff
@@ -61,8 +61,7 @@ public static void cleanup(OvalRootType root, OsFamily osFamily, String osVersio
             root.getDefinitions().removeIf(def -> def.getId().contains("unaffected"));
         }
 
-        // Debian OVAL files could contain patch definitions, but we're only interested in vulnerability definitions
-        if (osFamily == OsFamily.DEBIAN) {
+        if (osFamily == OsFamily.DEBIAN || osFamily == OsFamily.UBUNTU) {
```

Comment:
Not sure what this in this osFamily. If it is the salt grain value, than also Ubuntu has "Debian" as osFamily.
Like Alma, Rocky and Liberty has "RedHat".

Reply:
If this is not the osFamily from salt grains, we may want to use a different name to prevent confusion

Reply:
Good point Michael, but I think I will make it in a follow-up PR, since it's also associated with a DB entity and I don't want to polute this PR